src/schemas/tdc_time/TdcTime.h

```diff
@@ -0,0 +1,29 @@
+
+#pragma once
+
+#include "../../FlatBufferCreator.h"
+#include "../../SchemaRegistry.h"
+#include "../../helper.h"
+#include <map>
+#include <string>
+
+namespace TdcTime {
+class Converter : public FlatBufs::FlatBufferCreator {
+public:
+  Converter() = default;
+  ~Converter() = default;
```

Comment:
No need for these two lines, the default constructor and destructor will be generated anyway.
If you have reason to keep them then the destructor should be marked `override`.